src/comps/pages/orgs/admin/PendingMember.tsx

```diff
@@ -83,39 +85,57 @@ const PendingMember = ({
         enqueueSnackbar("User rejected!", { variant: "success" });
     };
 
+    useEffect(() => {
+        if (auto) {
+            handleApprove();
```

Comment:
Wouldn't this mean that a club admin has to navigate to the members list to approve it? If we are storing an auto status it could and should be checked by the `join-organization` organization function (and also skip stuff like sending "New Join Request" emails, instead just sending the acceptance message from `approve-member` [feel free to move shared code into `_shared/` rather than copy pasting)

Reply:
Ah, I see that this is meant to be used for "Approve All" - smart way to reuse code!